external/test-runtime/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "xunit.console.netcore": "1.0.3-prerelease-00921-01",
+    "xunit.runner.console": "2.2.0-beta2-build3300",
```

Comment:
Why not move this down to the net463 specific dependencies instead? Now that we are restoring for two different frameworks it might be good to move all the framework specific dependencies down to the frameworks section, and only keep here the shared ones, like the test data.